common/src/main/java/com/microsoft/identity/common/internal/commands/parameters/BrokerSilentTokenCommandParameters.java

```diff
@@ -89,7 +89,11 @@ public void validate() throws ArgumentException {
                     "mCallerPackageName", "Caller package name is not set"
             );
         }
-        if (SdkType.MSAL == getSdkType() &&
+        /**
+         * The check for SdkType.MSALCPP is added to ensure
+         * the uniformity of logic with SdkType.MSAL
+         */
```

Comment:
I believe this javadoc needs to be updated......and actually probably needs to be a comment instead of javadoc